indico/modules/rb/views/user/reservations.py

```diff
@@ -94,8 +89,10 @@ def __init__(self, rh, menu_item, **kwargs):
         self._menu_item = menu_item
```

Comment:
Overriding it directly seemed a bit confusing to me, as the value is coming from the RH. I though overriding it with a method/property and then returning the stored value would be more explicit and easier to understand. Not that sure now, though...